src/core/locator/qgslocatormodel.h

```diff
@@ -62,7 +63,9 @@ class CORE_EXPORT QgsLocatorModel : public QAbstractTableModel
       ResultFilterPriority SIP_MONKEYPATCH_COMPAT_NAME(ResultFilterPriorityRole), //!< Result priority, used by QgsLocatorProxyModel for sorting roles.
       ResultScore SIP_MONKEYPATCH_COMPAT_NAME(ResultScoreRole), //!< Result match score, used by QgsLocatorProxyModel for sorting roles.
       ResultFilterName SIP_MONKEYPATCH_COMPAT_NAME(ResultFilterNameRole), //!< Associated filter name which created the result
-      ResultFilterGroupSorting SIP_MONKEYPATCH_COMPAT_NAME(ResultFilterGroupSortingRole), //!< Group results within the same filter results
+      ResultFilterGroupSorting SIP_MONKEYPATCH_COMPAT_NAME(ResultFilterGroupSortingRole), //!< Deprecated since QGIS 3.40 no longer used
```

Comment:
```suggestion
      ResultFilterGroupSorting SIP_MONKEYPATCH_COMPAT_NAME(ResultFilterGroupSortingRole), //!< \deprecated QGIS 3.40. No longer used.
```

See https://github.com/qgis/QGIS/pull/58552